### PR TITLE
docs: update postgres docs with sample configuration [skip ci]

### PR DESCRIPTION
**Pull request by asvdf**
#### Pull Request Checklist

- [ ] All changes are Go version 1.16 compliant
- [ ] Your PR passes all CI checks. If a check cannot be passed for a justifiable reason, that reason must be stated in the commit message and PR description.
- [ ] If this is your first time contributing, we recommend you read the [Code Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
   - [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
   - [ ] Commits have a logical structure according to [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
- [ ] For new code: Code is accompanied by tests which exercise both the positive and negative (error paths) conditions (if applicable)
- [ ] For bug fixes: If possible, code is accompanied by new tests which trigger the bug being fixed to prevent regressions
- [ ] Any new logging statements use an appropriate subsystem and logging level
- [ ] For code and documentation: lines are wrapped at 80 characters (the tab character should be counted as 8 characters, not 4, as some IDEs do per default)
- [ ] A description of your changes [should be added to running the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes) for the milestone your change will land in, or,
- [ ] If a PR only fixes a trivial issue, such as updating documentation on a small scale, fix typos, or any changes that do not modify the code, the commit message should end with [skip ci] to skip the CI checks.

### CODE REVIEW

**Comment by asvdf** [comment]
Following https://github.com/lightningnetwork/lnd/pull/5982
I couldn't edit the commit title+message from web GUI (which is all I use) so I edited the documentation file again also following the suggestion to disable database open timeout, and created a new PR after closing the previous one.

File: docs/postgres.md [hunk 1]
```diff
@@ -6,11 +6,8 @@ describes how it can be configured.
 
 ## Building LND with postgres support
 
-To build LND with postgres support, include the following build tag:
-
-```shell
-⛰  make tags="kvdb_postgres"
-```
+To build LND with postgres support and for your specific platform, edit https://github.com/lightningnetwork/lnd/blob/1abe1da5b3a0e914b8c0f0d837fc0c17bed74a88/make/release_flags.mk#L41 by adding the `kvdb_postgres` to the end of the line.
```
Comment (guggero) [inline docs/postgres.md:9]: This is now done by default. Perhaps the command should just say `make release-install` which includes all required tags.

File: docs/postgres.md [hunk 2]
```diff
@@ -29,3 +26,12 @@ LND is configured for Postgres through the following configuration options:
   database, user and password.
 * `db.postgres.timeout=...` to set the connection timeout. If not set, no
   timeout applies.
+
+Example as follows:
+```
+[db]
+db.backend=postgres
+db.postgres.dsn=postgresql://dbuser:dbpass@127.0.0.1:5432/dbname
+db.postgres.timeout=0
```
Comment (guggero) [inline docs/postgres.md:35]: :+1:

**Review by guggero — CHANGES_REQUESTED** [review]
Your commit structure is all over the place... This can't be fixed through the UI though. Should I include your (squashed) commits in https://github.com/lightningnetwork/lnd/pull/6080?

**Comment by asvdf** [comment]
> Your commit structure is all over the place... This can't be fixed through the UI though. Should I include your (squashed) commits in #6080?

Yes, please do, thank you for your patience and sorry for my incompetence, I'm not a GitHub person!

**Comment by guggero** [comment]
Replaced by https://github.com/lightningnetwork/lnd/pull/6080 to speed up review process. Original commit author was preserved, thank you for your contribution.

**Pull request closed without merging**